package.json

```diff
@@ -45,6 +45,7 @@
     "depd": "^1.0.0",
     "inflection": "^1.6.0",
     "lodash": "^4.17.4",
+    "loopback-component-storage": "^3.2.0",
```

Comment:
This looks wrong to me. It should be the application who is deciding what version of loopback-component-storage it wants to use, not loopback-datasource-juggler.

Perhaps you should move this line to "devDependencies", to allow you to load the connector from your new test?